src/specs/components/blockquote/demo/data.js

```diff
@@ -3,4 +3,7 @@ module.exports = {
   citation:
     'An interconnected grid will help deliver the ultimate goal of the Energy Union, to make sure affordable, secure and sustainable energy, and also growth across the EU.',
   author: 'President Juncker',
+  image: {
+    path: 'https://inno-ecl.s3.amazonaws.com/media/examples/example-image3.jpg',
```

Comment:
don't be have another example image to use for this..? :) something with a european flag or so..?

Reply:
I need a 1:1 ratio or around there, i will try to find something else

Reply:
i like this image better.. :)

Reply:
fyi, we have a 1:1 image available: https://inno-ecl.s3.amazonaws.com/media/examples/example-image-square.jpg

Reply:
even better ;)